chrome/content/zotero/searchDialog.js

```diff
@@ -43,6 +43,8 @@ function doLoad()
 	document.getElementById('search-name').value = io.dataIn.name;
 
 	document.addEventListener('dialogaccept', doAccept);
+	// Focus the first focusable node in the dialog
+	Services.focus.moveFocus(window, sbc, Services.focus.MOVEFOCUS_FORWARD, 0);
```

Comment:
This line doesn't seem necessary? The Name input field gets focused for me automatically when editing a saved search, at least on macOS.

It would be better for the text to get selected, though.

Reply:
Good point! It does look like the first focusable field in dialogs gets selected (though I can't easily identify where exactly that happens). In any case, I swapped explicit focusing of the name field for selecting all text